examples/frontend/xtask/src/main.rs

```diff
@@ -0,0 +1,49 @@
+use anyhow::Result;
+use std::process;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+struct Opt {
+    #[structopt(subcommand)]
+    cmd: Command,
+}
+
+#[derive(StructOpt)]
+enum Command {
+    Build(xtask_wasm::Build),
+    StartServer(xtask_wasm::DevServer),
+    Serve(xtask_wasm::DevServer),
+    Watch(xtask_wasm::Watch),
+}
+
+fn main() -> Result<()> {
+    env_logger::builder()
+        .filter(Some("xtask"), log::LevelFilter::Trace)
+        .init();
+
+    let mut build_command = process::Command::new("cargo");
+    build_command.args(["run", "--example", "xtask", "--", "build"]);
```

Comment:
:thinking: that doesnt look good. Maybe try to make another workspace for the entire example?